content/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling.adoc

```diff
@@ -0,0 +1,141 @@
+continuous improvement system using open source software for both microservices
+and monolithic applications. He demonstrated how to
+create or import projects, and have them automatically build, run
+system and integration tests, stage, and finally deploy. He also showed to
+manage and update those deployed applications using continuous
+delivery practices.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-jenkins-at-redhat-final-final.pdf[*Download Presentation*]
+
+link:https://vimeo.com/173353537[*Watch the Video Demo from the Presentation*]
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/320px-CloudBees_official_logo.png[role=left]
+
+
+link:https://github.com/carlossg[Carlos Sanchez] of
+link:http://cloudbees.com[CloudBees] discussed
+link:https://www.cloudbees.com/scaling-jenkins-docker-swarm-kubernetes-or-mesos[*Scaling Jenkins with Docker: Swarm, Kubernetes or Mesos?*]
+He compared various Docker Swarm, Apache Mesos, and Kubernetes in terms of their
+ability to dynamically scale in Jenkins by running jobs inside containers.
+He also discussed the pros and cons, best practices, level of Jenkins support for each
+of these technologies.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-scaling_jenkins_with_docker_swarm_kubernetes_or_mesos_.pdf[*Download Presentation*]
+
+---
+
+link:https://www.cloudbees.com/so-you-want-build-worlds-biggest-jenkins-cluster[*So, You Want to Build the World's Biggest Jenkins Cluster?*]
+link:https://github.com/stephenc[Stephen Connolly] of
+link:http://cloudbees.com[CloudBees] showed all about it.  He started
+real world results realized by Jenkins users who have built large clusters.
+Next, he showed experiments around scaling some individual sub-components of Jenkins in
+isolation to see what challenges have been faced when integrated. Finally,
+he arrived at recipes for building Jenkins clusters with different scaling capabilities and
+making existing Jenkins clusters more efficient.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-soyouwanttobuildtheworldslargestjenkinscluster_final.pdf[*Download Presentation*]
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/splunk-logo-300x100.gif[role=right]
+
+
+link:https://www.linkedin.com/in/bill-houston-4909412[Bill Houston] and
+link:https://www.linkedin.com/in/aliraza82[Ali Raza] of
+link:https://www.splunk.com/[Splunk]
+gave a talk in two parts,
+link:https://www.cloudbees.com/jenkins-splunk-and-splunking-jenkins[*Jenkins at Splunk and Splunking Jenkins*]
+They first showed how Splunk uses Jenkins to implement their end-to-end CI system.
+They discussed features and design goals, challenges they encountered, and how they addressed
+these challenges.
+In the second part, they showed how to use the Jenkin Splunk plugin to gather
+test results and Jenkins environment data, and deliver it to a Splunk indexer for analysis and presentation.
```

Comment:
This "In the second" part sentence is seems kind of run-on, or directionless, to me.

Perhaps breaking it into two, e.g.:

> In the second part of the presentation Bill and Ali showed some of the features of the Jenkins Splunk plugin. Using the plugin, they gathered test results and environment data from Jenkins and delivered it to a Splunk indexer for future analysis and presentation.